pkg/menu/menu_test.go

```diff
@@ -0,0 +1,66 @@
+package menu
+
+import (
+	ui "github.com/gizak/termui/v3"
+	"strconv"
+	"testing"
+)
+
+func Test_newParagraph(t *testing.T) {
+	if err := ui.Init(); err != nil {
+		t.Fatal(err)
+	}
+	defer ui.Close()
+	const testText = "newParagraph test"
+	p := newParagraph(testText, false, 0, 50, 3)
+	if testText != p.Text {
+		t.Errorf("incorrect value for p.Text. got: %v, want: %v", p.Text, testText)
+	}
+}
+
+func pressKey(ch chan ui.Event, input []string) {
+	var key ui.Event
+	for _, id := range input {
+		key = ui.Event{
+			Type: ui.KeyboardEvent,
+			ID:   id,
+		}
+		ch <- key
+	}
+}
+
+func Test_GetInput(t *testing.T) {
+	if err := ui.Init(); err != nil {
+		t.Fatal(err)
+	}
+	defer ui.Close()
```

Comment:
nit: maybe add some newlines between your lines here and some of the other places where you have a lot of lines - just makes it easier to read :)